PWG/EMCAL/EMCALbase/AliTrackContainer.cxx

```diff
@@ -628,6 +629,37 @@ PWG::EMCAL::AliEmcalTrackSelResultHybrid::HybridType_t AliTrackContainer::GetHyb
   return hybridDefinition;
 }
 
+Bool_t AliTrackContainer::CheckArrayConsistency() const {
```

Comment:
I don't see where this function is called - did you just add it as a testing utility?

Reply:
Exactly, this is a test function. It was used to investigate the problems Leticia mentioned in a different thread.

I would like to keep it in the compiled code as part of a future unit test, and in order to have access to both arrays to has to be a member function as direct access to at least one of the arrays is prohibited (I would actually prefer prohibiting direct access to both arrays - that's the task of the container). Array consistency is important as it will create problems when accessing tracks with indices.

Reply:
Okay, thanks for the clarification!